lib/main_containers.pm

```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright 2021 SUSE LLC
+# Copyright 2022 SUSE LLC
```

Comment:
AFAIK it does not work that way , you suppose to extend not replace 
```suggestion
# Copyright 2021-2022 SUSE LLC
```
obviously same approach should be applied in all places where you updated copyright , don't want to spam PR with similar suggestions ...